rag_experiment_accelerator/evaluation/plain_metrics.py

```diff
@@ -1,153 +1,163 @@
 import evaluate
-import textdistance
-from fuzzywuzzy import fuzz
-
-algorithms = textdistance.algorithms
+from rapidfuzz import fuzz
+from rapidfuzz import distance as d
```

Comment:
Can we use full names instead of d and alg for better readability? (otherwise looks good)

Reply:
done